frontend/src/Album/EpisodeDetailsModalContentConnector.js

```diff
@@ -2,13 +2,13 @@ import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { createSelector } from 'reselect';
-import { clearReleases } from 'Store/Actions/releaseActions';
+import { cancelFetchReleases, clearReleases } from 'Store/Actions/releaseActions';
 import { toggleEpisodeMonitored } from 'Store/Actions/episodeActions';
 import createEpisodeSelector from 'Store/Selectors/createEpisodeSelector';
 import createArtistSelector from 'Store/Selectors/createArtistSelector';
 import episodeEntities from 'Album/episodeEntities';
 import { fetchTracks, clearTracks } from 'Store/Actions/trackActions';
-import { fetchTrackFiles, clearTrackFiles } from 'Store/Actions/trackFileActions';
+// import { fetchTrackFiles, clearTrackFiles } from 'Store/Actions/trackFileActions';
```

Comment:
Do we want to take this out before merge?

Reply:
Fixed